test/js-unit/recrasBookingSpec.js

```diff
@@ -32,6 +32,106 @@ describe('RecrasBooking', () => {
         });
     });
 
+    describe('amountsValid', () => {
+        let rb;
+        let inputLessThanMinimum;
+        let inputMoreThanMaximum;
+
+        beforeEach(() => {
+            let mainEl = document.createElement('div');
+            rb = new RecrasBooking(new RecrasOptions({
+                element: mainEl,
+                recras_hostname: 'demo.recras.nl',
+            }));
+
+            inputLessThanMinimum = document.createElement('input');
+            inputLessThanMinimum.value = 1;
+            inputLessThanMinimum.dataset.packageId = '1';
+
+            inputMoreThanMaximum = document.createElement('input');
+            inputMoreThanMaximum.value = 4;
+            inputMoreThanMaximum.dataset.packageId = '2';
+
+            let inputBookingSize = document.createElement('input');
+            inputBookingSize.id = 'bookingsize';
+            inputBookingSize.value = 1;
+
+            mainEl.appendChild(inputLessThanMinimum);
+            mainEl.appendChild(inputMoreThanMaximum);
+            mainEl.appendChild(inputBookingSize);
+        });
+
+        it('handles minimum amount', function() {
+            const pack = {
+                regels: [
+                    {
+                        id: 1,
```

Comment:
Does this number refer to line 49 (https://github.com/Recras/online-booking-js/pull/6/files#diff-30ce72ce21bf413a142429b0cfd21854R49) ? If so, it's probably better to let both reference the same object value 👍